README.md

```diff
@@ -4,13 +4,53 @@
 
 A web scraper tool to get data for evaluating Wellcome impact.
 
+## Installation
+
+### Common steps
+Install Postgresql (https://www.postgresql.org/) and create a `wsf_scraping` database
```

Comment:
Reminder: we'll soon want this to be automatically done within a `docker-compose.yaml`.

Reply:
We already have a docker-compose file, but it needs a serious update, I think 😄